### DISCUSSION TITLE
Add feature to move cursor programmically

### DISCUSSION THREAD

**Discussion by Pzixel** [hecrj/iced#866] [General]
I'm currently stuck without a function like `move_mouse_cursor_to(point: Point)` or  `move_mouse_cursor_to(element: Element)` 
I'd really like to have one in this awesome project.

If it exists somewhere here and I missed it please write in comments how can I do this in current version.

Thanks for your time.

**Comment by valpackett** [comment]
What is your use case for it? It's not a great way to do… anything, really. e.g. Wayland does not have a way to do this at all. If you want to be able to lock the pointer inside of an area (and possibly get relative motion like for a 3D camera), that's the abstraction that should be exposed by the GUI library (iced).

**Comment by Pzixel** [comment]
In my case I had a canvas and I wanted to have a button "center on canvas" which had to move cursor to well the center the canvas.
